### DISCUSSION TITLE
Will Base UI show demos with Pigment CSS in the future?

### DISCUSSION THREAD

**Discussion by tc** [mui/base-ui#608] [Discussions]
We are looking at component frameworks for our web app and want to support SSR/SSG.

I saw Material UI v6 supports Pigment CSS but wondering if Base UI will make the transition?

**Comment by colmtuite** [comment]
Base UI is an unstyled library, it doesn't use any styling engine. But we'll add Pigment CSS to the docs to show you how to use Base UI together with Pigment CSS.
